python/ray/rllib/optimizers/multi_gpu.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from ray.rllib.optimizers.optimizer import Optimizer
+
+
+class LocalMultiGPUOptimizer(Optimizer):
```

Comment:
Can you make sure to signal somehow this is not functional yet? Users will enable `config["multi_gpu_optimize"]` and it will fail in the `Optimizer` class.